BOOKING_SERVICE/main.py

```diff
@@ -21,136 +21,132 @@
     result = resource.resource_read_specific_id(id_resource)
     return jsonify(result)
 
 
 # Get a Resource by Name of the Database of Booking
-@app.route('/GetResourceByName', methods=['GET'])
-def get_resource_by_name():
+@app.route('/resource/name/<string:resource_name>', methods=['GET'])
+def get_resource_by_name(resource_name):
     resource = Resource()
-    result = resource.resource_read_specific_name(request.json)
+    result = resource.resource_read_specific_name({"name":str(resource_name)})
     return jsonify(result)
 
 
 # Update a Resource by Id of the Database of Booking
-@app.route('/UpdateResource/<string:id_resource>', methods=['PUT'])
+@app.route('/resource/<string:id_resource>', methods=['PUT'])
 def update_resource(id_resource):
     resource = Resource()
     result = resource.resource_update(id_resource, request.json)
     return jsonify(result)
 
 
 # Delete a Resource by Id of the Database of Booking
-@app.route('/DeleteResource/<string:id_resource>', methods=['DELETE'])
+@app.route('/resource/<string:id_resource>', methods=['DELETE'])
 def delete_resource(id_resource):
     resource = Resource()
     result = resource.resource_delete(id_resource)
     return jsonify(result)
 
 
 # Create a Person into the Database of Booking
-@app.route('/CreatePerson', methods=['POST'])
+@app.route('/person', methods=['POST'])
 def create_person():
     person = Person()
     result = person.person_create(request.json)
     return jsonify(result)
 
 
 # Get all Persons of the Database of Booking
-@app.route('/GetPersonAll', methods=['GET'])
+@app.route('/person', methods=['GET'])
 def get_person_all():
     person = Person()
     result = person.person_read_all()
     return jsonify(result)
 
 
 # Get a Person by Id of the Database of Booking
-@app.route('/GetPersonById/<string:id_person>', methods=['GET'])
+@app.route('/person/id/<string:id_person>', methods=['GET'])
 def get_person_by_id(id_person):
     person = Person()
     result = person.person_read_specific_id(id_person)
     return jsonify(result)
 
 
 # Get a Person by Name of the Database of Booking
-@app.route('/GetPersonByName', methods=['GET'])
-def get_person_by_name():
+@app.route('/person/name/<string:person_name>', methods=['GET'])
+def get_person_by_name(person_name):
     person = Person()
-    result = person.person_read_specific_name(request.json)
+    result = person.person_read_specific_name({"person_full_name": str(person_name)})
     return jsonify(result)
 
 
 # Update a Person by Id of the Database of Booking
-@app.route('/UpdatePerson/<string:id_person>', methods=['PUT'])
+@app.route('/person/<string:id_person>', methods=['PUT'])
 def update_person(id_person):
     person = Person()
     result = person.person_update(id_person, request.json)
     return jsonify(result)
 
-
-@app.route('/DeletePerson/<string:id_person>', methods=['DELETE'])
+# Delete a Person by Id of the Database of Booking
+@app.route('/person/<string:id_person>', methods=['DELETE'])
```

Comment:
Are you sure that only /person/<string: .... > will be enough for know that its a booking process?  If you do, don't worry and its ok